packages/ethereum-contracts/contracts/superfluid/Superfluid.sol

```diff
@@ -304,6 +304,10 @@ contract Superfluid is
         emit SuperTokenLogicUpdated(token, newLogic);
     }
 
+    function changeSuperTokenAdmin(ISuperToken token, address newAdmin) external onlyGovernance {
```

Comment:
should the admin be the manager here when it's set? test cases should rehearse  that transition of power over time too.

Reply:
okay, so the `changeAdmin` is correct then. nvm